network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -260,6 +261,30 @@ public void svcTest() {
         }
     }
 
+    @Test
+    public void testSvcRemove() {
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+            Network network = NetworkStorageTestCaseFactory.create(service.getNetworkFactory());
+            service.flush(network);
+        }
+
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+            Map<UUID, String> networkIds = service.getNetworkIds();
+            assertEquals(1, networkIds.size());
+            Network readNetwork = service.getNetwork(networkIds.keySet().stream().findFirst().get());
+            assertEquals(1, readNetwork.getStaticVarCompensatorCount());
+            readNetwork.getStaticVarCompensator("SVC2").remove();
```

Comment:
Maybe it can be checked that the equipment was correctly removed from the network before the flush. (Eventually add this check to all other equipments removal tests)